lib/ZwaveDevice.js

```diff
@@ -23,18 +23,25 @@ const commandClassParsers = {
 // TODO measure_pressure capability parser
 
 /**
- * @extends Homey.Device
- * @desc {@link https://developer.athom.com/docs/apps/Device.html#getSetting Device settings}
- * used by system capabilities:
- * - `invertWindowCoveringsDirection {boolean}` - Used by several windowcoverings capabilities,
+ * @classdesc
+ * Extends {@link https://apps-sdk-v3.developer.homey.app/Device.html Homey.Device}
+ *
+ * {@link https://apps.developer.homey.app/the-basics/devices/settings Device settings} used by
+ * system capabilities:
+ * - `invertWindowCoveringsDirection` type `checkbox`, Used by several windowcoverings capabilities,
  *    if true it will invert the up/down direction
- * - `invertWindowCoveringsTiltDirection {boolean}` - Used by several windowcoverings
+ * - `invertWindowCoveringsTiltDirection` type `checkbox`, Used by several windowcoverings
  *    capabilities, if true it will invert the tilt direction
- * @property {string} thermostatSetpointType - The 'Setpoint Type' used in the
- * THERMOSTAT_SETPOINT commandclass for the target_temperature capability
+ * @hideconstructor
  */
 class ZwaveDevice extends Homey.Device {
 
+  /**
+   * The 'Setpoint Type' used for the target_temperature capability.
+   * @type {string}
+   */
+  thermostatSetpointType = 'Heating 1';
```

Comment:
Not sure this is needed, it was merely added as documentation backup for: https://github.com/athombv/node-homey-zwavedriver/blob/master/lib/system/capabilities/target_temperature/THERMOSTAT_SETPOINT.js#L16

Reply:
Not sure I understand, you suggest to just not document that property?

The code change is only so that the JSDoc actually shows `thermostatSetpointType` as a property of `ZwaveDevice`, it's supposed to not affect the behaviour of the code.

Reply:
No, we should document it, but now it receives a default value at two places (in ZwaveDevice.js and in the system/capability handler). Preferably only the latter should set the default.

Reply:
I'm not sure about preferring "defaulting" it in the capability handler though.
I personally prefer setting it in the constructor over doing the `|| 'Heating 1'`; less cases to account for elsewhere and gives me a place to add JSDoc/types.